### PR TITLE
Update to 19.17 

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

**Comment by jschueller** [comment]
@conda-forge-admin, please rerender

**Comment by jschueller** [comment]
this is ready to go @patricksnape @aldanor

File: recipe/bld.bat [hunk 1]
```diff
@@ -1,4 +1,6 @@
 
+git clone --depth 1 https://github.com/davisking/dlib.git dlib_git && cd dlib_git
```
Comment (aldanor) [inline recipe/bld.bat]: What's this doing here?...

There's `meta.yaml` for pulling source, at specific git revisions / tags / branches.

Comment (jschueller) [reply recipe/bld.bat]: I was just experimenting, I'll remove it, sorry

**Comment by jschueller** [comment]
@aldanor I removed the commit, it can be merged (the ci will give be green in a few minutes)

**Comment by patricksnape** [comment]
Thanks @jschueller - I looked at this the other day but came up completely stumped!

**Pull request merged**